### PR TITLE
Update to v1.5

### PR DESCRIPTION
**Pull request by Lonsfor**
#32

### CODE REVIEW

**Comment by Lonsfor** [comment]
works

File: org.srb2.SRB2Kart.desktop [hunk 1]
```diff
@@ -8,8 +8,12 @@ Exec=srb2kart
...
 Categories=Game;
-Actions=OpenGL;
+Actions=OpenGL;software;
 
 [Desktop Action OpenGL]
 Name=OpenGL
 Exec=srb2kart -opengl
+
+[Desktop Action Software]
+Name=Solfware
```
Comment (ActualMandM) [inline org.srb2.SRB2Kart.desktop:18]:
```suggestion
Name=Software
```

**Review by ActualMandM — CHANGES_REQUESTED** [review]

**Comment by Eoin-ONeill-Yokai** [comment]
I'm having an issue where the game stalls after the window opens and is unresponsive. The last valid line from console output reads:

```
HMS: connecting 'https://ms.kartkrew.org/ms/api/rules?v=2.2'...
```

**Comment by Lonsfor** [comment]
are you using opengl on an nvidia gpu @Eoin-ONeill-Yokai ?

**Comment by Lonsfor** [comment]
there seems to be some issues with nvidia gpus in upstream so i think that this merge should be left alone for a day or two.

**Comment by Eoin-ONeill-Yokai** [comment]
> are you using opengl on an nvidia gpu @Eoin-ONeill-Yokai ?

@Lonsfor I'm actually not -- I'm using an AMD GPU on XServer. Should I try opening with the `-opengl` flag?

**Comment by Lonsfor** [comment]
sure

**Comment by Eoin-ONeill-Yokai** [comment]
Tried it just now -- hasn't changed. I cross checked it with my other flatpak installation and it seems to be working there. 

I might install it on aur to see if I'm having the same problem.

**Comment by HybridEidolon** [comment]
> edit2: Yeah, aur works fine. Notably, the game seems to be experiencing a lot of network issues atm.

We will be releasing another version soon with an emergency fix soon. The problem is related to the new NAT holepunching code. Just figured I should give another head's up.

**Comment by Lonsfor** [comment]
@HybridEidolon while you are here, are we allowed to distribute the assets directly? i mostly ask because having people not being able to install until someone notices is annoying.

**Comment by Lonsfor** [comment]
> Yeah, aur works fine.

whack. i wonder if with the http-ms-startup-print fix it might be fixed anyway or if it need a push in the runtime.

**Comment by HybridEidolon** [comment]
> @HybridEidolon while you are here, are we allowed to distribute the assets directly? i mostly ask because having people not being able to install until someone notices is annoying.

This should be fine (cc @TehRealSalt). It's not really feasible to give permission since a number of the assets are derived from other games. To that end if it's possible for the flatpak to download the assets from the github release when running it for the first time, that might be better, but I can't think of any trivial way to set that up atm.

**Comment by Eoin-ONeill-Yokai** [comment]
FWIW, it is now working for me on the latest testing release. Good job!

**Comment by Lonsfor** [comment]
whack

**Comment by Lonsfor** [comment]
Well now that issue is resolved this is now for sure ready to be merged.

**Pull request merged**